platformio.ini

```diff
@@ -350,7 +334,7 @@ framework     = arduino
 board         = disco_f765vg
```

Comment:
https://github.com/platformio/platform-ststm32/tree/develop/boards

This board doesnt exist at all, is that a typo ?

Reply:
`remram_v1` works fine for our purposes. But it's still not fully compiling due to an issue with `FastIOPortMap` not being defined, due to `GPIO_TypeDef` not being defined, I suppose.